packages/fx-core/templates/core/v3Migration/migrationReport.md

```diff
@@ -1,6 +1,10 @@
 # Upgrade Summary
 
-Congratulations! Your project has been upgraded to work with latest Teams Toolkit. You can visit https://aka.ms/teamsfx-v5.0-guide to learn new features in Teams Toolkit. And visit https://aka.ms/teams-toolkit-5.0-upgrade to learn more about the upgrade.
+Congratulations! You can continue your development now. Your project has been upgraded to work with latest Teams Toolkit.
```

Comment:
latest -> the latest
as Chaoyi also recommended, move the "what's next" up, to be right after this first paragraph. if the upgrade wiki has details on what's next after upgrade.

Reply:
Updated the wording.
Actually the sentence `You can continue your development now.` is the "what's next". Users do not need to do anything immediately after upgrade.